content/participants/ttjc@ashishvb/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Ashish Bhogesara FullStack Developer
+type: "participant"
+name: "Ashish Bhogesara"
+portfolio: "https://ashishvb.com/"
+github-link: "https://github.com/ashishvb"
+linkedin-link: "https://in.linkedin.com/in/ashish-bhogesara-8338a4186"
+looking-for: "job"
+description: "REACT | ANGULAR | IONIC | NODEJS | MONGODB"
+---
+
+Hello, I'm Ashish Bhogesara let me walk you through my intro.
+
+### introduction
+
+I am self taugh software developer, passed out in 2018 as mechanical engineer. 
```

Comment:
Hey! @ashishvb a small mistake here:
* taught

Reply:
Hey! @khusharth  bro, thanks a ton for bringing it to notice. have made the necessary changes. Please go through it once again. Sorry for the rework.